src/JanusFtl.cpp

```diff
@@ -1003,12 +1003,11 @@ ConnectionResult JanusFtl::onOrchestratorStreamRelay(ConnectionRelayPayload payl
                 };
         }
 
-        if (relayClients.count(payload.ChannelId) <= 0)
-        {
-            relayClients.insert_or_assign(payload.ChannelId, std::list<ActiveRelay>());
-        }
-        relayClients.at(payload.ChannelId).emplace_back(payload.ChannelId, payload.TargetHostname,
-            std::move(relayClient));
+        relayClients[payload.ChannelId].push_back(ActiveRelay {
+            .ChannelId = payload.ChannelId,
+            .TargetHostname = payload.TargetHostname,
+            .Client = std::move(relayClient),
```

Comment:
Clang really doesn't like this emplace_back due to something with the relayClient unique_ptr. After an hour of reading I'm still not sure why but I'm willing to give up the teeny tiny amount of perf to get twice as fast compiles.